src/components/landing/landing.js

```diff
@@ -19,115 +19,76 @@ class Landing extends Component {
+              <h3>Search Events</h3>
             </div>
-            <div className="icons-wrapper">
-              <div>
-                <i className="fas fa-hand-pointer"></i>
-                <h3>Search Events</h3>
-              </div>
-              <div>
-                <i className="fas fa-user-plus"></i>
-                <h3>Create Your Own Event</h3>
-              </div>
-              <div>
-                <i className="fas fa-search"></i>
-                <h3>Find Events Partners</h3>
-              </div>
-              <div>
-                <i className="far fa-smile"></i>
-                <h3>Get Connected</h3>
-              </div>
-              <div>
-                <i className="fas fa-users"></i>
-                <h3>Have Fun Together</h3>
-              </div>
+            <div>
+              <i className="fas fa-user-plus"></i>
+              <h3>Create Your Own Event</h3>
             </div>
-            <AuthComponent onlyAnonymous>
-              <div className="text-center">
-                <ModalWind
-                  renderButton={action => (
-                    <button
-                      className="btn btn-warning"
-                      onClick={() => action()}
-                      type="button"
-                    >
-                      Join EventsExpress
-                    </button>
-                  )}
-                />
-              </div>
-            </AuthComponent>
-          </article>
-          {events.length &&
-            events.map(block => this.renderCarouselBlock(block))}
-          {/* <article className="events-article">
-            <div className="row">
-              <div className="col-md-10">
-                <h3>Upcoming events</h3>
-              </div>
-              <div style={{ textAlign: "right" }} className="col-md-2">
-                <a href="/home/events">Explore more events</a>
-                <EventMatrix />
+            <div>
+              <i className="fas fa-search"></i>
+              <h3>Find Events Partners</h3>
+            </div>
+            <div>
+              <i className="far fa-smile"></i>
+              <h3>Get Connected</h3>
+            </div>
+            <div>
+              <i className="fas fa-users"></i>
+              <h3>Have Fun Together</h3>
+            </div>
+          </div>
+          <AuthComponent onlyAnonymous>
+            <div className="text-center">
+              <ModalWind
+                renderButton={action => (
+                  <button
+                    className="btn btn-warning"
+                    onClick={() => action()}
+                    type="button"
+                  >
+                    Join EventsExpress
+                  </button>
+                )}
+              />
+            </div>
+          </AuthComponent>
+        </article>
+        {items.length !== 0 && (
+          <>
+            <article className="events-article">
+              <div className="row">
```

Comment:
Extra divs, Move styles into article tag

Reply:
I deleted extra divs and created one className for it, but I think this is a temporary solution. Work on this landing page still needs to be done and will be done in the future